specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerSecurityUserConfiguration.json

```diff
@@ -0,0 +1,996 @@
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NetworkManagerSecurityConfigurationImport"
+            },
+            "description": "Import Security configuration parameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Post successful.",
+            "schema": {
+              "$ref": "#/definitions/SecurityConfigurationImportResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Import NSG Rules": {
+            "$ref": "./examples/NetworkManagerSecurityConfigurationImport.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/securityUserConfigurations/{configurationName}/evaluateImport": {
+      "post": {
+        "tags": [
+          "SecurityUserConfigurations"
+        ],
+        "operationId": "SecurityUserConfigurations_EvaluateImport",
+        "description": "The operation to evaluate import NSG to security user configurations.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SecurityConfigurationParameter"
+          },
+          {
+            "$ref": "#/parameters/ListTopParameter"
+          },
+          {
+            "$ref": "#/parameters/ListSkipTokenParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NetworkManagerSecurityConfigurationImport"
+            },
+            "description": "Import security configuration parameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Post successful.",
+            "schema": {
+              "$ref": "#/definitions/SecurityConfigurationRuleListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
```

Comment:
as in other file - if your post needs to be pageable you need to add support for it - the default impl will do a GET on the nextlink